src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,27 @@
-export const TodoList = () => {};
+import React from 'react';
+import cn from 'classnames';
+import { TodoInfo } from '../TodoInfo';
+import { UserInfo } from '../UserInfo';
```

Comment:
The `UserInfo` component should not directly import data or utilities like `getUser`. Instead, ensure that all necessary data is prepared in the `App` component and passed down as props. This aligns with the checklist item: 'Prepare data in one place (App component) and pass it to child components. Don't use `import` keyword in non-root components.'